components/mitc_service/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GIT
   remote: https://github.com/ideacrew/aca_entities.git
-  revision: 9926268f4b31bda907a8c0e19f0d020627ed0714
-  branch: release_0.8.0
+  revision: 238d79f8a1c769fb6a517de5eb3a7ab44592f335
+  branch: release_0.7.0
```

Comment:
we don't want to be merging in any reference to the 0.7.0 version of aca entities